libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -757,6 +757,16 @@ void SRV_Channels::set_output_to_trim(SRV_Channel::Function function)
     }
 }
 
+void SRV_Channels::set_output_scaled_to_trim(SRV_Channel::Function function)
+{
+    for (uint8_t i=0; i<NUM_SERVO_CHANNELS; i++) {
+        if (channels[i].function == function) {
+            channels[i].set_output_pwm(channels[i].servo_trim);
+            set_output_scaled(function, channels[i].range_from_pwm() * channels[i].high_out);
```

Comment:
On problem here is that you could have several output channels set to a given function. However, there is only one scaled output for each function. So, you will end up with the scaled output that represents the last channel in this loop.